packages/labs/gen-wrapper-react/goldens/test-element-a/package.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "test-element-a-react",
+  "type": "module",
+  "scripts": {
+    "build": "tsc",
+    "build:watch": "tsc --watch"
+  },
+  "version": "1.0.0",
+  "dependencies": {
+    "test-element-a": "^1.0.0",
+    "@lit-labs/react": "^1.0.4"
+  },
+  "peerDependencies": {
+    "react": "^17.0.2",
+    "@types/react": "^17.0.19"
+  },
+  "devDependencies": {
+    "typescript": "^4.3.5"
```

Comment:
to be fixed in the test element: We should use `~` for TypeScript constraints since they break at minor versions.

Reply:
Fixed and update goldens.

I realized the test package didn't actually specify `typescript` as a devDep, so added it and changed the generator to use the version from the source package if it exists.